source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "This Month in Solidus: October 2020"
+date: 2020-11-10
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/shared/images/solidus-monthly-update.png
+description: >
+  Dive in and see everything that happened in the Solidus world in October 2020
+---
+
+Hello everyone, and welcome to This Month in Solidus — October 2020! Let's dive in to what's new with the platform.
+
+### Extension updates for October
+
+- [**Solidus Dev Support**](https://github.com/solidusio/solidus_dev_support) has seen quite a few updates for this month - there's now a boilerplate for extension configuration files, we've added configuration for `github_changelog_generator` (ensuring consistent changelogs across extensions), and made a few more minor changes and bugfixes.
+
+- [**Solidus Content**](https://github.com/solidusio-contrib/solidus_content) now supports DatoCMS!
+
+- [**Solidus Subscriptions**](https://github.com/solidusio-contrib/solidus_subscriptions) now integrates with Churn Buster, event emissions and tracking has been improved, and a tab has been added to the admin user view that displays existing subscriptions for any given user. Handy!
+
+- [**Solidus PayPal Commerce Platform**](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform) has had a few minor bugfixes, but more importantly you're now able to display PayPal's credit messaging to your users before checkout, which is shown to increase conversion rates and average order value. Version 0.2.1 is now live!
+
+- [**Solidus PayPal Braintree**](https://github.com/solidusio/solidus_paypal_braintree) has also had PayPal's credit messaging added (when PayPal is being used and the checkout flow is set to `checkout`), in addition to a whole lot of minor improvements and an update to PayPal's new SDK (which allows the new PayPal credit options like Pay in 4 to be available). Version 1.0.0 is now live!
+
+### Core updates this October
+
+Here's what happened in October:
+
+- [Updated permission set admin tab display.](https://github.com/solidusio/solidus/pull/3783) Previously, if a user had access to only a subset of the menu items under 'settings', it would prevent the 'settings' tab from showing up. This bugfix - by Elia Schito - fixes that issue!
+- [Updated incorrect state generation for Italy.](https://github.com/solidusio/solidus/pull/3722) In any instance where a `Carmen::Subregion` existed for a country, we would use the subregion as the countries states. However, in some instances, this is incorrect. In Italys case, we need to be using the subregions subregions. There are more countries with this issue - but this PR provides the framework to fix those countries as they are identified. Sean Denny - that's me - is responsible for this fix!
```

Comment:
I think here you meant "country's" instead of "countries" and "Italy's" instead of "Italys."